uqcsbot/scripts/mock.py

```diff
@@ -29,40 +29,39 @@ def get_nth_most_recent_message(channel_id: str, message_index: int):
 
 
 def mock_message(message: str):
-    '''
-    Given a message, will return the mocked version of it. This involves
-    randomly varying the case of each letter in the message.
+    """
+    Given a message, will return the mocked version of it.
+    This involves randomly varying the case of each letter in the message.
 
     Example:
       Input: "Mitch, you're acting very immature and should probably stop"
       Output: "MitCh, YoU’RE aCTing vERy ImMatUrE aNd sHOuld pRObAbLy stOp"
 
     See: http://knowyourmeme.com/memes/mocking-spongebob
-    '''
+    """
     return ''.join(choice((c.upper, c.lower))() for c in message)
 
 
 def is_number(message):
-    '''
-    Tries to coerce the message to an integer. If successful, return True-- else
-    returns False.
-    '''
+    """
+    Tries to coerce the message to an integer. If successful, return True else returns False.
+    """
     try:
         int(message)
-    except:
+    except Exception:
```

Comment:
Better to use ValueError rather than Exception

Reply:
Fixed.